### PR TITLE
rc1 RELEASE.md

### PR DESCRIPTION
**Pull request by av8ramit**
Updating the authors in RELEASE.md and updating new bug fixes.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -68,38 +68,16 @@
 * Multiple tfdbg bug fixes:
   * Fixed Windows compatibility issues.
   * Command history now persists across runs.
+* Bug fix in graph validation related to `tf.while_loops`.
+* Java Maven fixes for bugs with Windows installation.
+* Rollback of adjust contract change.
```
Comment (gunan) [inline RELEASE.md]: No need to mention this.

File: RELEASE.md [hunk 2]
```diff
@@ -68,38 +68,16 @@
...
+* Initial release of installation guides for Java, C, and Go.
```
Comment (gunan) [inline RELEASE.md]: This is a major thing to mention.

**Comment by av8ramit** [comment]
I signed it!

File: RELEASE.md [hunk 3]
```diff
@@ -68,38 +68,16 @@
...
...Sawruk, Jiaming Liu, Jihun Choi, @jiqiu, Joan Thibault, John C F, Jojy George Varghese, Jon Malmaud, Julian Berman, Julian Niedermeier, Junpeng Lao, Kai Sasaki, @Kankroc, Karl Lessard, Kyle Bostelmann, @Lezcano, Li Yi, Luo Yun, @lurker, Mahmoud-Abuzaina, Mandeep Singh, Marek Kolodziej, Mark Szepieniec, Martial Hue, Medhat Omr, Memo Akten, Michael Gharbi, MichaëL Defferrard, Milan Straka, @MircoT, @mlucool, Muammar Ibn Faisal, Nayana Thorat, @nghiattran, Nicholas Connor, Nikolaas Steenbergen, Niraj Patel, Niranjan Hasabnis, @Panmari, Pavel Bulanov, Philip Pries Henningsen, Philipp Jund, @polonez, Prayag Verma, Rahul Kavi, Raphael Gontijo Lopes, @rasbt, Raven Iqqe, Reid Pryzant, Richard Shin, Rizwan Asif, Russell Kaplan, Ryo Asakura, RüDiger Busche, Saisai Shao, Sam Abrahams, @sanosay, Sean Papay, @seaotterman, @selay01, Shaurya Sharma, Sriram Narayanamoorthy, Stefano Probst, @taknevski, @tbonza, @teldridge11, @terrytangyuan, Tim Anglade, Tomas Reimers, Tomer Gafner, Valentin Iovene, Vamsi Sripathi, Viktor Malyi, Vit Stepanovs, Vivek Rane, Vlad Firoiu, @wangg12, @will, Xiaoyu Tao, Yaroslav Bulatov, Yi Liu, Yuan (Terry) Tang, @Yufeng, Yuming Wang, Yuxin Wu, Zafar Takhirov, Ziming Dong
```
Comment (gunan) [inline RELEASE.md]: To remove:
Damien Martin-Guillerez is a googler
Daniel Mane is a googler,
@drpngx is a googler
Chi Zeng is a googler
Remove `@terrytangyuan` as he is already mentioned by real name later.

Also, let's linewrap this. no need to have it as a single gigantic line.

**Review by gunan — CHANGES_REQUESTED** [review]

**Review by yifeif — CHANGES_REQUESTED** [review]
Remove Damien Martin-Guillerez, Daniel Mané,  @drpngx from external contributor list?

**Review by gunan — APPROVED** [review]

**Review by yifeif — APPROVED** [review]

File: RELEASE.md [hunk 4]
```diff
@@ -68,38 +69,39 @@
 * Multiple tfdbg bug fixes:
   * Fixed Windows compatibility issues.
   * Command history now persists across runs.
+* Bug fix in graph validation related to `tf.while_loops`.
```
Comment (caisq) [inline RELEASE.md]: Increase indent.

Comment (av8ramit) [inline RELEASE.md]: Done.

**Pull request merged**